fictadvisor-web/src/app/(main)/account/components/general-tab/components/contacts-block/components/ContactItem.tsx

```diff
@@ -23,7 +22,7 @@ const ContactItem: FC<ContactProps> = ({
   const { user } = useAuthentication();
   const handleDeleteClick = async () => {
     await UserAPI.deleteContact(user.id, id);
-    refetchContacts();
+    await refetchContacts();
```

Comment:
there is a queryClient.invalidateQueries(), so you dont need to pass it as a prop